app/_src/kubernetes-ingress-controller/references/feature-gates.md

```diff
@@ -44,6 +44,20 @@ Features that reach GA and become stable are removed from this table, but they c
 {:.important}
 >**Important:** To avoid disruption to your services consider not using features until they have reached GA status. 
 
+### Using feature gates
+
+To enable feature gates you can provide the `--feature-gates` flag when launching KIC, or set the `CONTROLLER_FEATURE_GATES` environment variable.
+
+Feature gates are a comma delimited set of `key=value` pairs. If you wanted to enable `CombinedRoutes` and `CombinedServices`, you'd set `CONTROLLER_FEATURE_GATES=CombinedRoutes=true,CombinedServices=true`
```

Comment:
```suggestion
Feature gates consist of a comma-delimited set of `key=value` pairs. For example, if you wanted to enable `CombinedRoutes` and `CombinedServices`, you'd set `CONTROLLER_FEATURE_GATES=CombinedRoutes=true,CombinedServices=true`.
```
Suggesting "consist of a" instead of "are a" for better subject-verb agreement.